### ISSUE TITLE
Records using absolute addressing failing with calls to `adsGetSymHandleByName`

### ISSUE THREAD

**Issue by klauer** [pcdshub/twincat-ads#8]
See: https://github.com/pcdshub/ads-ioc/pull/77

Resulting in the following:
```
2021/06/17 18:01:30.962 [../adsAsynPortDriver.cpp:978] adsAsynPortDriver:drvUserCreate: Parameter created: "ADSPORT=350/POLL_RATE=1/.ADR.16#F200,16#100,32,19?" (index 549).
2021/06/17 18:01:30.973 [../adsAsynPortDriver.cpp:2876] adsAsynPortDriver:adsGetSymHandleByName: Create handle for .ADR.16#F200,16#100,32,19 failed with: ADSERR_DEVICE_SYMBOLNOTFOUND (0x710)
```

**Comment by mcb64** [comment]
Actually, I think the answer is... don't do that.

In updateParamInfoWithPLCInfo, we should have:
if (!paramInfo->isAdrCommand) {
      adsReleaseSymbolicHandle(paramInfo,true); //try to delete
      status=adsGetSymHandleByName(paramInfo);
      if(status!=asynSuccess){
        return asynError;
      }
}

Then, it looks like adsAddDataCallback/adsAddToBulkRead do the right things with absolute addresses, and we avoid the error message and return.

**Comment by klauer** [comment]
That seems logical to me!

I'm a bit curious why it nukes the handle and then gets it again - or is the release just a no-operation at startup?

**Comment by mcb64** [comment]
I think this is an artifact of the days before we started crashing the IOC whenever the PLC goes down.

I that case, I think the free was just an abundance of caution to make sure if we disconnected, the PLC really *did* crash.  Because if it didn't, we'd happily get another handle, and if we're not careful, run out of handles.  (I could be wrong, but I have some vague memory that you *can* have two handles with the exact same address.)
